### DISCUSSION TITLE
GEN_ENS_PROD_MASK_GRID, GEN_ENS_PROD_MASK_POLY

### DISCUSSION THREAD

**Discussion by BinbinZhou-NOAA** [dtcenter/METplus#1922] [Configuration] [answered]
I can not find  GEN_ENS_PROD_MASK_GRID and  GEN_ENS_PROD_MASK_POLY in the METplus document.
So I assume that the ensemble product output are still over the input ensemble grid.
Just want to confirm this.

Thanks!

Binbin

**Comment by jprestop** [comment, marked as answer]
Hi @BinbinZhou-NOAA.

> I assume that the ensemble product output are still over the input ensemble grid.

I believe that is correct.  Looking at the [Practical Information section of the GenEnsProd documentation](https://met.readthedocs.io/en/develop/Users_Guide/gen-ens-prod.html#practical-information), I see the following:

> All input gridded model and climatology datasets must be on the same grid. However, users may leverage the automated regridding feature in MET if the desired output grid is specified in the configuration file.

As I understand it, users could change the output grid by using the automated regridding feature in MET.

@JohnHalleyGotway, please correct me if I am wrong.

**Reply by BinbinZhou-NOAA** [reply]
Julie,

  Thanks!

Binbin

**Reply by jprestop** [reply]
Hi @BinbinZhou-NOAA.  Do you have any further questions?  If not, I’d like to mark this discussion as being answered and “lock it” to prevent future posts. That’s how we encourage users to create new discussions for new questions. But I wanted to give you an opportunity to comment on it before doing so. Please feel free to select one of the responses as being the best answer to your original question. That’ll help future users with similar questions find answers.

**Comment by BinbinZhou-NOAA** [comment]
No further questions, please close this discussion.

Thanks!

Binbin

On Tue, Nov 29, 2022 at 11:28 AM jprestop ***@***.***> wrote:

> Hi @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA>. Do you have any
> further questions? If not, I’d like to mark this discussion as being
> answered and “lock it” to prevent future posts. That’s how we encourage
> users to create new discussions for new questions. But I wanted to give you
> an opportunity to comment on it before doing so. Please feel free to select
> one of the responses as being the best answer to your original question.
> That’ll help future users with similar questions find answers.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/1922#discussioncomment-4265918>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIHQSGW7ED3NIWML3TWKYVIZANCNFSM6AAAAAARZX6ZZM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683
